onnx_chainer/functions/math.py

```diff
@@ -212,6 +212,15 @@ def convert_Sqrt(func, opset_version, input_names, output_names,
         return onnx_helper.make_node('Sqrt', input_names, output_names),
 
 
+def convert_RsqrtGPU(func, opset_version, input_names, output_names,
+                     context, parameters):
+    gb = onnx_helper.GraphBuilder()
+    sqrt_out = gb.op('Sqrt', input_names)
+    one = context.add_const(np.array(1, dtype=func.inputs[0].dtype), 'one')
```

Comment:
I think `Reciprocal` is a better op here

Reply:
thanks!